src/Features/Blockcore.Features.Wallet/Exceptions/MiningNotification.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Blockcore.Features.Wallet.Exceptions
+{
+    public static class MiningNotification
```

Comment:
this should go in the mining feature not wallet feature

Reply:
Done